src/components/Reviews/ReviewsPage.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import ReviewCard from "./ReviewCard";
+import * as api from "../../api";
```

Comment:
It might be nice just to import the function you need here rarther than everything from the api file